custom_components/bambu_lab/pybambu/models.py

```diff
@@ -1683,7 +1683,6 @@ class AMSInstance:
     serial: str
     sw_version: str
     hw_version: str
-    model: str
```

Comment:
Why is this removed? It is needed to be able to do `coordinator.get_model().ams.data[index].model == "AMS 2 Pro"` in the definition.py right?